.github/workflows/ci.yml

```
@@ -101,6 +101,23 @@ jobs:
       - name: Machete
         uses: bnjbvr/cargo-machete@main
 
+  # Checks for duplicate version of package
+  cargo-vendor:
+    name: Cargo Vendor
+    runs-on: 8core_ubuntu_latest_runner
```

This is our paid runner which we use for the slow parts to speed them up. So let's use the free runner for this job.
```suggestion
    runs-on: ubuntu-latest
```